dgbpy/hdf5.py

```diff
@@ -657,17 +656,21 @@ def getInfo( filenm, quick ):
     retinfo.update({plfdictstr: odhdf5.getText(info,'Model.Type')})
   if  odhdf5.hasAttr(info,versionstr):
     retinfo.update({versiondictstr: odhdf5.getText(info,versionstr)})
-  if img2img & isclassification:
-    groups = retinfo[exampledictstr].keys()
-    ret = list()
-    for groupnm in groups:
-      try:
-          grp = h5file[groupnm]
-          for inpnm in grp:
-            if np.any(np.array(grp[inpnm][ydatadictstr])==-1):
-              retinfo.update({withunlabeleddictstr: True})
-      except:
-        pass
+  hasunlabels = False
```

Comment:
The condition if img2img and isclassification should stay. None of this should be done otherwise